src/netconfig.cpp

```diff
@@ -304,15 +292,15 @@ static void cmdVlan(Dbus& bus, Arguments& args)
 static const Command commands[] = {
     {"show", nullptr, "Show current configuration", cmdShow},
     {"reset", nullptr, "Reset configuration to factory defaults", cmdReset},
-    {"mac", "MAC", "Set MAC address", cmdMac},
+    {"mac", "{INTERFACE} MAC", "Set MAC address", cmdMac},
     {"hostname", "NAME", "Set host name", cmdHostname},
     {"gateway", "IP", "Set default gateway", cmdGateway},
-    {"ip", "[VLANID] {add|del} IP[/MASK GATEWAY]", "Add or remove static IP address", cmdIp},
-    {"dhcp", "[VLANID] {enable|disable}", "Enable or disable DHCP client", cmdDhcp},
+    {"ip", "{INTERFACE} {add|del} IP[/MASK GATEWAY]", "Add or remove static IP address", cmdIp},
+    {"dhcp", "{INTERFACE} {enable|disable}", "Enable or disable DHCP client", cmdDhcp},
     {"dhcpcfg", "{enable|disable} {dns|ntp}", "Enable or disable DHCP features", cmdDhcpcfg},
-    {"dns", "[VLANID] {add|del} [static] IP", "Add or remove DNS server", cmdDns},
-    {"ntp", "[VLANID] {add|del} IP", "Add or remove NTP server", cmdNtp},
-    {"vlan", "{add|del} ID", "Add or remove VLAN", cmdVlan},
+    {"dns", "{INTERFACE} {add|del} [static] IP", "Add or remove DNS server", cmdDns},
+    {"ntp", "{INTERFACE} {add|del} IP", "Add or remove NTP server", cmdNtp},
```

Comment:
Neither VLANID nor INTERFACE belong to DNS/NTP configurations. Both `dns` and `ntp` are global settings, not per-interface. Just like `gateway` and `hostname`.

Reply:
I have to bind DNS/NTP to a network interface, it is by design of OpenBMC's network manager.
I can use eth0 in Nicole, but what about Vegman?

Reply:
Ok, current implementation uses eth0 for manipulating with DNS/NTP server list.

Reply:
Good. I think that it's another misunderstanding in upstream OpenBMC of how routing in Linux kernel works, just like it was with default gateway earlier. If it doesn't work for us on Vegman with two interfaces on BMC, then we'll think further what to do.